easybuild/framework/easyconfig/tweak.py

```diff
@@ -166,7 +166,7 @@ def __repr__(self):
     for (key, val) in tweaks.items():
 
         if isinstance(val, list):
-            regexp = re.compile(r"^\s*%s\s*=\s*(.*)$" % key, re.M)
+            regexp = re.compile(r"^(?P<key>\s*%s)\s*=\s*(?P<val>.*)$" % key, re.M)
```

Comment:
No, including the `\s*` into the group, and using the group rather than hard replacing the key, that fixes it.